ota-plus-web/app/com/advancedtelematic/controllers/Application.scala

```diff
@@ -45,6 +47,20 @@ object UiToggles {
   }
 }
 
+final case class EnvironmentMode(
```

Comment:
Looks fine, I just find it slightly easier to read as:
```
final case class EnvironmentMode(isDev: Boolean, isProd: Boolean)
object EnvironmentMode {
  def apply(env: Environment): EnvironmentMode =
    EnvironmentMode(env.mode == Dev, env.mode == Prod)
}
```
In any case, we should probably move this to `Data.scala` in this package and import it here, to try to keep all the models in one place.

Reply:
Also: good use of the companion object 👌

Reply:
Done, thanks